django/db/migrations/recorder.py

```diff
@@ -18,30 +19,39 @@ class MigrationRecorder:
     If a migration is unapplied its row is removed from the table. Having
     a row in the table always means a migration is applied.
     """
+    _migration_class = None
 
-    class Migration(models.Model):
-        app = models.CharField(max_length=255)
-        name = models.CharField(max_length=255)
-        applied = models.DateTimeField(default=now)
+    def __init__(self, connection):
+        self.connection = connection
 
-        class Meta:
-            apps = Apps()
-            app_label = "migrations"
-            db_table = "django_migrations"
+    @classproperty
+    def migration(cls):
```

Comment:
And if we just call this `Migration` (with the capital `M`) can we avoid the other changes later in the file?

Reply:
Done!